apps/editing-toolkit/editing-toolkit-plugin/site-editor/index.php

```diff
@@ -86,5 +61,5 @@ function is_site_editor_active() {
 	 *
 	 * @param bool true if Site Editor should be enabled, false otherwise.
 	 */
-	return apply_filters( 'a8c_enable_core_site_editor', false );
+	return is_readable( get_stylesheet_directory() . '/block-templates/index.html' );
```

Comment:
the `apply_filters` was determining eligibility based on a blog sticker.  Its replacement is essentially the same as the `gutenberg_is_fse_theme` function used in core Gutenberg, except here we must use `get_stylesheet_directory` instead of `STYLESHEETPATH` as it will be more accurate in the dotcom setting.

Reply:
Could we just switch to `gutenberg_is_fse_theme`? (or is `STYLESHEETPATH` a big limitation?)

Reply:
Is this because we are applying filters to this function on Dotcom side? If so would it make more sense to use `get_stylesheet_directory` in core too?

Reply:
> Could we just switch to gutenberg_is_fse_theme? 

That is what I tried at first, but it didn't work.  When I re-wrote the same function here, the linter notified me that I should be using `get_stylesheet_directory` instead of `STYLESHEETPATH`... and then it started working. 🤣 

> Is this because we are applying filters to this function on Dotcom side? 

Im not entirely sure.

> If so would it make more sense to use get_stylesheet_directory in core too?

Maybe?  Although core seems pretty consistent about defining `STYLESHEETPATH` as get_stylesheet_directory() and using `STYLESHEETPATH` instead.

Reply:
Im wondering if the reason is that `get_stylesheet_directory` is redefined on dotcom to get the correct path for simple sites.

Reply:
It looks like functions using `gutenberg_is_fse_theme` were taken off earlier actions and added to another in D52323-code for the same issue experienced when trying to use that here.  Although that action doesn't seem to work in this case.  Either way I don't think using `get_stylesheet_directory` is a blocker here, but it would be nice to be able to use the function as supplied by gutenberg at some point. 🤔